manager.py

```diff
@@ -143,8 +143,11 @@ def get_service_logger_names(self, service_name, add_level):
                                     add_level=add_level)
         return self._service_manager.execute_request(service_name, request)
 
-    @staticmethod
-    def get_log_entries(name, entries_count=-1, level=None, component=None):
+    def _service_list(self):
+        return self._service_manager.instances.configuration.get_children()
+
+    def get_log_entries(
```

Comment:
I had to remove the `@staticmethod` from this. Do you see any problem with this or was it static before just because it could be? @f1401martin

Reply:
no problem, it was static because it could be